include/ur_api.h

```diff
@@ -8314,7 +8314,7 @@ urCommandBufferRetainExp(
 ///     - ::UR_RESULT_ERROR_OUT_OF_HOST_MEMORY
 UR_APIEXPORT ur_result_t UR_APICALL
 urCommandBufferReleaseExp(
-    ur_exp_command_buffer_handle_t hCommandBuffer ///< [in][release] Handle of the command-buffer object.
+    ur_exp_command_buffer_handle_t &hCommandBuffer ///< [in][release] Handle of the command-buffer object.
```

Comment:
This header is a C API, there can not be use of C++ references in it. Additionally, this header is generated from a spec description in yaml files which reside in `scripts/core/*.yml`.